test/built-ins/Atomics/wait/bigint/negative-timeout-agent.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2018 Rick Waldron. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-atomics.wait
+description: >
+  Test that Atomics.wait times out with a negative timeout
+includes: [atomicsHelper.js]
+features: [Atomics, BigInt, SharedArrayBuffer, TypedArray]
+---*/
+
+$262.agent.start(`
+  $262.agent.receiveBroadcast(function(sab, id) {
+    const i64a = new BigInt64Array(sab);
+    $262.agent.report(Atomics.wait(i64a, 0, 0, -5)); // -5 => 0
+    $262.agent.leaving();
+  });
+`);
+
+const i64a = new BigInt64Array(
+  new SharedArrayBuffer(BigInt64Array.BYTES_PER_ELEMENT * 8)
+);
+
+$262.agent.broadcast(i64a.buffer);
+$262.agent.sleep(10);
```

Comment:
I wonder if the agents might be slow enough so we need to keep the sleep time higher for stability?

Reply:
I'm working on moving them all to `waitUntil`